dynatrace_base/tox.ini

```diff
@@ -28,6 +27,3 @@ commands = flake8 .
 [flake8]
 exclude = .eggs,.tox,build
 max-line-length = 120
-
-[pytest]
-addopts = -p no:warnings
```

Comment:
a new annoying warning now there?)

Reply:
No warnings in dynatrace base library, so we not need to hide them.